src/engine/renderer/tr_shader.cpp

```diff
@@ -6172,7 +6189,7 @@ shader_t       *R_FindShader( const char *name, shaderType_t type, int flags )
 	if ( flags & RSF_2D )
 	{
 		imageParams_t imageParams = {};
-		imageParams.bits = bits;
+		imageParams.bits = bits | IF_SRGB;
```

Comment:
Isn't that the opposite of what you want? sRGB to linear conversion should be done when sampling a texture used for 3D rendering, but not when sampling one used for 2D rendering.

Reply:
Yes, that was a leftover. Now removed.

Anyway since `GL_FRAMEBUFFER_SRGB` was disabled when those images were processed, the sRGB property was likely ignored.